src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpCodeActions.cs

```diff
@@ -523,7 +523,6 @@ static void Main(string[] args)
             var expectedItems = new[]
             {
                 "Rename 'P2' to 'Foober'",
-                "Generate type 'Foober'",
```

Comment:
This change is due to the fact that the normal priority bucket now has one lesser code action (for `Suppress or Configure issues`), so the below sum is now 3 instead of 4 and we are able to inline the Generate type nested actions and move them to top level lightbulb. https://github.com/dotnet/roslyn/blob/ae8beb0ee14397f6da2d8099a2964c441a86fd1f/src/Features/Core/Portable/UnifiedSuggestions/UnifiedSuggestedActionsSource.cs#L595-L605

Reply:
that seems reasonable.